python/matlab_converter.py

```diff
@@ -0,0 +1,17 @@
+from utils import SimResult
+import numpy as np 
+import scipy.io
+import argparse
+
```

Comment:
Maybe add something like. It's a nice practice but not required
```suggestion
if __name__ == "__main__"
```